app/javascript/src/components/Notes/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+import { Button } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
+
+import Card from "components/Notes/Card";
+import { NOTES_LIST } from "components/Notes/constants";
+import SidePanel from "components/Notes/SidePanel";
+
+const NoteListing = () => (
+  <div className="flex flex-row">
+    <SidePanel title="Notes" />
+    <div className="flex flex-col">
+      <Header
+        actionBlock={<Button className="mx-2" label="Add Note +" />}
+        menuBarToggle={function noRefCheck() {}}
+        title="All Notes"
+        searchProps={{
+          onChange: function noRefCheck() {},
+          value: "",
+        }}
+      />
+      <Container>
+        {NOTES_LIST.map(card => (
```

Comment:
- First of all why's this named `NOTES_LIST` over `NOTES`? How does suffixing with `LIST` be of any benefit over here?
- You are mapping through `NOTES_LIST` but the iterated element is named `card`. How does that make sense?